cli/flags.go

```diff
@@ -0,0 +1,14 @@
+package cli
+
+import "github.com/urfave/cli/v2"
+
+type ViamStringFlag struct {
+	cli.StringFlag
+}
+
+// Names have to be overwritten to prevent required flag errors from using aliases in its message.
+// This returns f.Name as the last member of Names().
+func (f ViamStringFlag) Names() []string {
+	names := append(f.Aliases, f.Name)
+	return cli.FlagNames(names[0], names[1:])
```

Comment:
[q] Why return f.Name as the last member (have aliases precede)? I assumed the point of this was to hide the `robot` aliases?

Reply:
for whatever reason, the error uses the last member of the Names() list, which would be an alias if we don't do it this way

Reply:
..strange, but sounds good to me.